src/main/java/gift/domain/product/dao/ProductDao.java

```diff
@@ -0,0 +1,71 @@
+package gift.domain.product.dao;
+
+import gift.domain.product.dto.ProductDto;
+import gift.domain.product.entity.Product;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import org.springframework.jdbc.core.simple.JdbcClient;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ProductDao {
+
+    private final JdbcClient jdbcClient;
+
+    public ProductDao(JdbcClient jdbcClient) {
+        this.jdbcClient = jdbcClient;
+    }
+
+    public Product insert(Product product) {
+        String sql = "INSERT INTO product (name, price, image_url) VALUES (?, ?, ?)";
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+
+        jdbcClient.sql(sql).param(product.getName()).param(product.getPrice()).param(product.getImageUrl())
```

Comment:
```suggestion
        jdbcClient.sql(sql)
        .param(product.getName())
        .param(product.getPrice())
        .param(product.getImageUrl())
```
한줄에 하나씩 작성한다면 훨씬 보기 좋을거 같습니다.

Reply:
메소드 체이닝에서 항상 줄 바꿈을 해야하나 말아야 하나 고민했는데 의견 감사합니다~!